os_builders/roles/nubes_bootcontext/files/nubes-boot.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=basic boot script
+After=network.service
+Before=sshd.service
+
+[Service]
+Environment= MY_ENVIRONMENT_VAR
```

Comment:
Does this need to be set, or is a leftover from the original docs?

Reply:
Left over, removed now